l10n_br_account_banking_payment_cnab/febraban/cnab_240/cnab_240.py

```diff
@@ -163,9 +180,62 @@ def remessa(self, order):
         :return:
         """
         self.order = order
+        banco_num = self.order.mode.bank_id.bank_bic
+        banco = self.get_cnab_bank(banco_num)
         self.arquivo = Arquivo(self.bank, **self._prepare_header())
+        codigo_evento = 1
+        evento = Evento(self.bank, codigo_evento)
+
+        count = 0
         for line in order.line_ids:
-            self.arquivo.incluir_cobranca(**self._prepare_segmento(line))
+
+            seg = self._prepare_segmento(line)
+            if int(banco_num) == 33:
+                seg_s = banco.registros.SegmentoS(**seg)
+            if seg_s.mensagem_recibo_pagador == 2 and count == 0:
```

Comment:
if the line `if int(banco_num) == 33` is false, then the `seg_s` variable is false and this line will raise `UnbountLocalError`.